torch/utils/cpp_extension.py

```diff
@@ -1682,7 +1715,9 @@ def object_file_path(source_file: str) -> str:
 
     objects = [object_file_path(src) for src in sources]
 
-    if IS_WINDOWS:
+    if is_standalone:
+        ldflags = extra_ldflags
+    elif IS_WINDOWS:
```

Comment:
nittiest of nits: it would feel better if it was:

```
if is_standalone:
  ...
else:
  if IS_WINDOWS:
    ...
  else:
    ...
```

as this communicates better that there aren't three cases, there are two cases, and a subdivision in the second case.

Reply:
Fair point, though my cyclomatic complexity alarm went off for the double nest. I wound up defining `SHARED_FLAG` since platform is import constant, and then just did `ldflags = ([] if is_standalone else SHARED_FLAG) + extra_ldflags`. A cynic might argue that I'm being too cute / python-y, but hopefully that conveys the intent of "either it's standalone, or slap a `--plays_well_with_others` stamp on it.)